web_src/js/components/DiffFileTreeItem.vue

```diff
@@ -119,6 +127,12 @@ export default {
   border-radius: 4px;
 }
 
+.item.filewrapper.active {
```

Comment:
Why not combine the three rules with the same content into one?
What I mean is a rule like
`.item.filewrapper.active, .item.filewrapper:hover, div.directory:hover{`

Reply:
done!